src/main/kotlin/no/nav/familie/ef/iverksett/infrastruktur/json/BrevJson.kt

```diff
@@ -10,3 +13,19 @@ data class BrevdataJson(
     val mottaker: String,
     val saksbehandler: String
 )
+
```

Comment:
pdf-en er en ByteArray mer enn String.

Så må vi vurdere hvordan dette skal løses for å unngå OOM:
- Sende med i JSON (krever ca. 10-20x minne mtp pdf-størrelse pga deserialisering)
- Sende som ekstra input i http-kallet som egen type (krever ca. 2x minne mtp pdf-størrelse) 
- Sende med referanse og "hente" pdf i iverksettingssteget - ekstra integrasjon.

Reply:
Satser på å bake inn pdf for brev i en MultipartFile som en "RequestPart" da ?